src/index.rs

```diff
@@ -39,7 +39,7 @@ pub struct VariableIndexEntry {
     is_constant: bool,
     /// the variable's datatype
     pub data_type_name: String,
-    /// the index of the member-variable in it's container (e.g. struct). defautls to 0 (Single variables)
+    /// the index of the member-variable in it's container (e.gxxx. struct). defautls to 0 (Single variables)
```

Comment:
i'm assuming you didn't mean to change this?

Reply:
well  ... :rofl: